### DISCUSSION TITLE
Preview links can be composed based on field data

### DISCUSSION THREAD

**Discussion by belafonte** [Cockpit-HQ/Cockpit#98] [Ideas]
Hey there,

I have another idea :).

With features like "draft mode" in frontend frameworks, it could be handy if preview links could have access to field data.
Since routes often include the id or a slug.

https://nextjs.org/docs/app/building-your-application/configuring/draft-mode

**Comment by aheinze** [comment]
Yes it is planned to support item data in preview link urls 👍

**Comment by belafonte** [comment]
Very cool. Looking forward to it! Thank you
